README.md

```diff
@@ -17,11 +17,69 @@
 
 ## About
 
+Piko Logger is a tiny logger for web environments. 
+
 ## Features
 
+- It's enabled/disabled through local storage
+- Can create an instance of a keyed logger
+- Log operations are called in functional way to avoid executions on non-activated log operations 
+- There are thwo log levels:
+    - Debug
+    - Error
+- Will format keyed loggers as: LOG_LEVEL | #KEY# | #MESSAGE#
+
 ## Technical features
 
 ## Usage
 
+### Enable/disble the logger
+By default the log level is set to off, so logs are disabled.
+There are two levels of enableing it:
+- Application level: 
+    
+    *"piko.level" = #logLevel*
+
+    ```code
+    window.localStorage.setItem('piko.level', 'debug')
+    ```
+- Keyed logger level:
+    
+    *"piko.level.#key" = #logLevel*
+
+    Example for a logger initialized with a key with value "exampleKey"
+    ```code
+    window.localStorage.setItem('piko.level.exampleKey', 'error')
+    ```
+*The keyed logger level overrides the application level, which in turn overrides the default logger level
+
+### Initialize the logger
+The logger should be initialized with a custom key to identify the logger. 
+```code
+import PikoLoggerInitializer from '@adv-ui/piko-logger'
```

Comment:
as the exported API is the PikoLogger already initialized, this should be changed to:

```code
import PikoLogger from '@adv-ui/piko-logger'
const logger = PikoLogger.logger(key)
```

Reply:
changed, thanks